src/integrant/core.cljc

```diff
@@ -212,20 +224,10 @@
             :config config
             :key key}))
 
-(defn- resolve-ref [config resolvef ref]
-  (let [[k v] (first (find-derived config (ref-key ref)))]
-    (resolvef k v)))
-
-(defn- resolve-refset [config resolvef refset]
-  (set (for [[k v] (find-derived config (ref-key refset))]
-         (resolvef k v))))
-
 (defn- expand-key [config resolvef value]
   (walk/postwalk
-   #(cond
-      (ref? %)    (resolve-ref config resolvef %)
-      (refset? %) (resolve-refset config resolvef %)
-      :else       %)
+   #(cond-> %
```

Comment:
Can you change this to an `if`? e.g.

```clojure
  #(if (reflike? %) (ref-resolve % config resolvef) %)
```

For simple checks I find that an `if` is easier to parse.